app/src/main/java/com/sunasterisk/movie19/ui/splashscreen/SplashScreen.kt

```diff
@@ -2,17 +2,24 @@ package com.sunasterisk.movie19.ui.splashscreen
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.os.Handler
 import android.view.WindowManager
 import com.sunasterisk.movie19.R
+import com.sunasterisk.movie19.ui.main.MainActivity
 
 class SplashScreen : AppCompatActivity() {
 
+    private val TIME = 1000L
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_splash_screen)
         getWindow().setFlags(
             WindowManager.LayoutParams.FLAG_FULLSCREEN,
             WindowManager.LayoutParams.FLAG_FULLSCREEN
         )
+        Handler().postDelayed({
+            startActivity(MainActivity.getIntent(this))
+        }, TIME)
```

Comment:
Đọc bài này và sửa lại nhé, 
https://medium.com/@ssaurel/create-a-splash-screen-on-android-the-right-way-93d6fb444857
không phải delay gì cả

Reply:
Uây, tự động có delay luôn hay chỉ là delay do khởi chạy MainActivity nhỉ

Reply:
```suggestion
        startActivity(MainActivity.getIntent(this))
```